rapid7/metasploit-framework
# Add initial pentesting docs section

Adding a new pentesting section to https://docs.metasploit.com

<img width="1153" alt="image" src="https://user-images.githubusercontent.com/60357436/164766024-e1e29d57-1d11-4175-8703-00b3852c1be5.png">

### Verification steps

Build the docs site locally and verify the new pages are available:

https://github.com/rapid7/metasploit-framework/blob/f737e78b876711f1a8d05809fd4632bc95181e65/docs/README.md
